tests/Microsoft.Fx.Portability.Tests/Microsoft.Fx.Portability.Tests.csproj

```diff
@@ -24,7 +21,7 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;FEATURE_XML_SCHEMA</DefineConstants>
```

Comment:
I switched it to reference the desktop version and there is a test for TargetMapper that relies on that if/def. In desktop it validates the target map file against the schema and provides a slightly more helpful error message than the portable one where schema validation is not available.